compute/instances/troubleshooting/fix-unreachable-focal-with-two-public-ips.mdx

```diff
@@ -0,0 +1,121 @@
+---
+meta:
+  title: Fix unreachable Ubuntu Focal Instance when it as two public IPv4 IPs.
+  description: This page helps you to recover or avoid unreachable Ubuntu 20.04 Focal Instance when it has two public IPv4 IP addresses.
+content:
+  h1: Fix unreachable Ubuntu Focal Instance when it as two public IPv4 IPs.
+  paragraph: This page helps you to recover or avoid unreachable Ubuntu 20.04 Focal Instance when it has two public IPv4 IP addresses.
+tags: ubuntu-focal focal 20.04 multiple ipv4 ip
+dates:
+  validation: 2024-07-23
+  posted: 2024-07-23
+categories:
+  - compute
+---
+
+When using an Ubuntu 20.04 Focal Instance with more than one routed IP address, the instance will become unreachable after a reboot. This may happen when you attach a second routed IPv4 or IPv6 IP address to the instance.
```

Comment:
```suggestion
When using an Ubuntu 20.04 Focal Instance with more than one routed IP address, the Instance will become unreachable after a reboot. This may happen when you attach a second routed IPv4 or IPv6 IP address to the Instance.
```